### DISCUSSION TITLE
Issue template is not opening

### DISCUSSION THREAD

**Discussion by jakbin** [community/community#12910] [Mobile]
When try to create an issue with template , but it is not working.

https://user-images.githubusercontent.com/65591973/158497859-7f2afe53-3b6d-4066-a1c4-23fdf3791fa1.mp4

**Comment by aconsuegra** [comment]
Thanks for the bug report, I can reproduce it on my end too. I'll create an issue to track it and hopefully we can ship a fix soon.

Edit: @jakbin Could you share the device and Android version that you are using?

**Reply by jakbin** [reply]
os sdk v30,

Samsung sm-M70F
